src/Nimb3s.Automaton.Api/Controllers/JobController.cs

```diff
@@ -21,6 +22,23 @@ public JobController(IMessageSession messageSession)
             this.messageSession = messageSession;
         }
 
+        // Gets Job status of job using the Job Id
+
+        [HttpGet("api/automaton/jobs/{jobId}")]
```

Comment:
bonus points for adding the appropriate metadata to the method. it'll help with generating the proper swagger documentation. something like this. you can reference the [swagger documentation](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-5.0&tabs=visual-studio):

```
        /// <summary>
        /// Creates a <see cref="JobCreatedModel"/> item.
        /// </summary>
        /// <remarks>
        /// Sample request:
        ///
        ///     POST /asfd
        ///     {
        ///     }
        /// </remarks>
        /// <response code="200">Returns ok when the automation job is reset to <see cref="JobStatusType.Created"/> or <see cref="JobStatusType.Started"/> </response>
        /// <response code="201">Returns the newly created <see cref="JobCreatedModel"/></response>
        /// <response code="400">If the item is not found</response> 
        /// 
        /// Get the job status of a job to the client

        [ProducesResponseType(StatusCodes.Status200OK)]
        [ProducesResponseType(StatusCodes.Status201Created)]
        [ProducesResponseType(StatusCodes.Status400BadRequest)]
```